homeassistant/components/smhi/weather.py

```diff
@@ -140,6 +140,9 @@ def __init__(
         )
         self._attr_condition = None
         self._attr_temperature = None
+        self._attr_wind_speed = None
+        self._attr_visibility = None
+        self._attr_pressure = None
```

Comment:
These attributes are initialized by the base class already.

https://github.com/home-assistant/core/blob/cdd5a5f68b3293c295c2230ac25676f51c5aed53/homeassistant/components/weather/__init__.py#L117-L126

Why do we need to do it again here?

Reply:
Not sure if something changed but now I get
```text
************* Module homeassistant.components.smhi.weather
Warning: homeassistant/components/smhi/weather.py:162:12: W0201: Attribute '_attr_wind_speed' defined outside __init__ (attribute-defined-outside-init)
Warning: homeassistant/components/smhi/weather.py:164:12: W0[20](https://github.com/home-assistant/core/runs/6925891566?check_suite_focus=true#step:7:21)1: Attribute '_attr_visibility' defined outside __init__ (attribute-defined-outside-init)
Warning: homeassistant/components/smhi/weather.py:165:12: W0201: Attribute '_attr_pressure' defined outside __init__ (attribute-defined-outside-init)
```

Reply:
Are you sure you're based on dev branch and not your Weather unit branch?

Reply:
I am but I wonder if my vscode didn't have a hiccup due to that work.
Probably went to fast on this. I will close it.